Diagnostic/diagnostic.py

```diff
@@ -695,6 +695,26 @@ def start_daemon():
         hutil.error("wait daemon start time out")
 
 
+def check_suspected_memory_leak(pid):
+    memory_leak_threshold_in_KB = 2000000  # Roughly 2GB
```

Comment:
Hosung, please open an issue to make "Amount of memory consumed by mdsd which will trigger a restart" a PublicConfig parameter. That should help me remember to do something about this.